### ISSUE TITLE
stackup.py math error for LEF58_SPACINGTABLE

### ISSUE THREAD

**Issue by ssteffl** [ucb-bar/hammer#523]
it uses >= instead of >. this will cause min-pitch wires to never be possible. min-width wires should match the "width_at_least": 0.0. 

src/hammer-tech/stackup.py line 176 is an example.

I started to fix it, but realized this was an issue in more places than i had realized, so i aborted.

refer to page 479 of lefdef 5.8 spec

**Comment by edwardcwang** [comment]
@jwright6323

**Comment by jwright6323** [comment]
Yeah, we talked about this in the meeting on Thursday. I recall deliberately using >=, so I don't think it's a bug but will look into it.
